src/test/java/com/wikia/webdriver/testcases/discussions/ReportingPostTests.java

```diff
@@ -0,0 +1,620 @@
+    PostsCreator postsCreator = postsListPage.getPostsCreatorDesktop();
+
+    PostEntity postEntity = addPostAndGoToUserPostsPage(postsListPage, postsCreator);
+    postData.put(POST_ON_DESKTOP_ON_USER_PAGE, postEntity.toData());
+
+    assertThatPostCanBeReported(postEntity);
+  }
+
+  @Test(groups = "discussions-loggedInUsersDesktopReporting",
+      dependsOnMethods = {
+          "userOnDesktopCanReportPostOnPostDetailsPage",
+          "anonUserOnDesktopCanNotSeeReportedPostOnPostDetailsPage",
+          "userOnDesktopCannotSeeReportedIndicatorOnPostsReportedByAnotherUserOnPostDetailsPageAndCanReportThatPost",
+          "moderatorOnDesktopCanApproveReportedPostOnPostDetailsPage"})
+  @Execute(asUser = User.USER)
+  @InBrowser(browser = Browser.FIREFOX, browserSize = DESKTOP_RESOLUTION)
+  public void userOnDesktopCannotReReportPostOnPostDetailsPage() {
+    final String postId = postData.get(POST_ON_DESKTOP_ON_POST_DETAILS).getId();
+
+    PostDetailsPage page = new PostDetailsPage().open(postId);
+
+    PostEntity postEntity = page.getPost().findPostById(postId);
+
+    assertThatPostIsNotReported(postEntity);
+    assertThatReportPostOptionIsNotAvailableFor(postEntity);
+  }
+
+  // Second user on desktop
+
+  @Test(groups = "discussions-loggedInUsersDesktopReporting",
+      dependsOnMethods = "userOnDesktopCanReportPostOnPostsListPage")
+  @Execute(asUser = User.USER_2)
+  @InBrowser(browser = Browser.FIREFOX, browserSize = DESKTOP_RESOLUTION)
+  public void userOnDesktopCannotSeeReportedIndicatorOnPostsReportedByAnotherUserOnPostsListPageAndCanReportThatPost() {
+    final String postId = postData.get(POST_ON_DESKTOP_ON_POSTS_LIST).getId();
+
+    PageWithPosts page = new PostsListPage().open();
+
+    assertThatPostReportedByOtherUserDoesNotHaveReportedIndicatorAndCanBeReportedByCurrentUser(page, postId);
+  }
+
+  @Test(groups = "discussions-loggedInUsersDesktopReporting",
+      dependsOnMethods = "userOnDesktopCanReportPostOnPostDetailsPage")
+  @Execute(asUser = User.USER_2)
+  @InBrowser(browser = Browser.FIREFOX, browserSize = DESKTOP_RESOLUTION)
+  public void userOnDesktopCannotSeeReportedIndicatorOnPostsReportedByAnotherUserOnPostDetailsPageAndCanReportThatPost() {
+    final String postId = postData.get(POST_ON_DESKTOP_ON_POST_DETAILS).getId();
+
+    PageWithPosts page = new PostDetailsPage().open(postId);
+
+    assertThatPostReportedByOtherUserDoesNotHaveReportedIndicatorAndCanBeReportedByCurrentUser(page, postId);
+  }
+
+  @Test(groups = "discussions-loggedInUsersDesktopReporting",
+      dependsOnMethods = "userOnDesktopCanReportPostOnUserPostsPage")
+  @Execute(asUser = User.USER_2)
+  @InBrowser(browser = Browser.FIREFOX, browserSize = DESKTOP_RESOLUTION)
+  public void userOnDesktopCannotSeeReportedIndicatorOnPostsReportedByAnotherUserOnUserPostsPageAndCanReportThatPost() {
+    final PostEntity.Data postData = this.postData.get(POST_ON_DESKTOP_ON_USER_PAGE);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename "postData" which hides the field declared at line 41. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AHiddenFieldCheck)